west.yml

```diff
@@ -98,7 +98,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 66db7c2a57ec1adfd3574169346ed3af9bf17c0b
+      revision: pull/118/head
```

Comment:
This commit is just changing west.yml, but the commit log says:

```
Adds flash partitions and chosen nodes to the mimxrt1050_evk_qspi
device tree to support mcuboot on the internal QSPI flash.

Also enables FlexSPI flash driver XIP mode support on this board to
support mcuboot.
```

This doesn't seem to be doing any of that; what am I missing here? It looks like you need to fix this commit to add those DT changes.

Reply:
You are right. At this point that commit message no longer makes sense. It probably is a consequence of this: https://github.com/zephyrproject-rtos/zephyr/pull/38872#issuecomment-964473636 , but I should have updated the commit message.

Reply:
@mbolivar-nordic , please, let me know if you find this commit message to be more suitable:

https://github.com/zephyrproject-rtos/zephyr/pull/38872/commits/873494b4bd4e6a2f9d1c7d519e35296d3591e2b6